dbos/_db_wizard.py

```diff
@@ -28,7 +28,10 @@ class DatabaseConnection(TypedDict):
     local_suffix: Optional[bool]
 
 
-def db_wizard(config: "ConfigFile", config_file_path: str) -> "ConfigFile":
+# This function first checks connectivity to the database configured in Transact
```

Comment:
This comment feels like it should be a doc string (https://peps.python.org/pep-0257/)